gui/adventure_module.py

```diff
@@ -12,41 +12,42 @@ def __init__(self):
         self.screen_height = config.getint("General", "screen_height")
         self.screen_size = (self.screen_width, self.screen_height)
         self.win = pygame.display.set_mode(self.screen_size)
-        self.background = pygame.transform.scale(pygame.image.load(os.path.join('..', 'assets', 'adventure_background.png')),
+        self.background = pygame.transform.scale(pygame.image.load(os.path.join('..', 'assets',
+                                                                                'adventure_background.png')),
                                                  self.screen_size)
 
         self.button_surface = pygame.image.load(os.path.join('..', 'assets', 'button.png'))
         self.button_surface = pygame.transform.scale(self.button_surface, (300, 100))
 
-        self.button_menu = Button(self.button_surface, self.screen_width / 2, 950, 700, 950, "menu")
-        self.button_beach = Button(self.button_surface, self.screen_width / 2, 150, 700, 150, "beach")
+        self.button_back = Button(self.button_surface, 600, 950, 600, 950, "back")
+        self.button_mountain = Button(self.button_surface, self.screen_width / 2, 150, 700, 150, "mountain")
         self.button_ruins = Button(self.button_surface, 1700, 350, 1700, 350, "ruins")
-        self.button_desert = Button(self.button_surface, 200, 300, 200, 300, "desert")
+        self.button_desert = Button(self.button_surface, 200, 300, 200, 300, "mountain")
```

Comment:
Why `self.button_desert` have `"mountain"` as an argument instead of `"desert"`?

Reply:
Good point. I got tangled up while trying to change all these names.